### ISSUE TITLE
mis-redirect?

### ISSUE THREAD

**Issue by r2evans** [dill/rdrr-redirect#1]
Seeing your tweet, I immediately downloaded it to try it ... perhaps that's sloppy/code-promiscuous of me.

I searched for boxplot and google provided: https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwjpq_TvrtHuAhWHUt8KHYBNB-MQFjAAegQIBBAC&url=https%3A%2F%2Fwww.rdocumentation.org%2Fpackages%2Fgraphics%2Fversions%2F3.6.2%2Ftopics%2Fboxplot&usg=AOvVaw2OMNvqbdmktQL0ypTE-Le5

When I clicked on it, the redirect was to https://rdrr.io/cran/graphics/man/boxplot.html, which does not work. :-(

(FF 85.0, win10)

Nice idea, I like it. What I've been doing (to avoid that site) is I have a FF bookmark:

```
Location:  https://www.google.com/search?client=firefox-b-1-d&q=site%3Astat.ethz.ch%2FR-manual%2FR-devel%2Flibrary%2F+%s
Keyword:   r/stat
```

In the FF url field, I type in `r/stat boxplot`, and it searches under `stat.ethz.ch` (which is usually austere help pages in fixed-width, more than good enough for most of the SO answers I use it for).

**Comment by dill** [comment]
You're right, that *should* work! I'll fix that and upload a new release. Thanks for reporting the bug!
